nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfigEdit.js

```diff
@@ -1082,6 +1116,179 @@ export function ApnResourcesEdit(props: Props) {
           ))}
         </List>
       </DialogContent>
+      <DialogActions>
+        <Button onClick={props.onClose} skin="regular">
+          Cancel
+        </Button>
+        <Button onClick={onSave} variant="contained" color="primary">
+          {props.isAdd ? 'Save And Continue' : 'Save'}
+        </Button>
+      </DialogActions>
+    </>
+  );
+}
+
+export function HeaderEnrichmentConfig(props: Props) {
+  const enqueueSnackbar = useEnqueueSnackbar();
+  const [error, setError] = useState('');
+  const ctx = useContext(GatewayContext);
+  const [heConfig, setHeConfig] = useState<gateway_he_config>(
+    props.gateway?.cellular.he_config || DEFAULT_HE_CONFIG,
+  );
+
+  const handleHEChange = (key: string, val) => {
+    setHeConfig({...heConfig, [key]: val});
+  };
+
+  const [
+    cellularConfig,
+    _setCellularConfig,
```

Comment:
minor nit: _setCellularConfig -> setCellularConfig

Reply:
I added this because setCellularConfig was never used and it raised flow errors